Gemfile

```diff
@@ -1,17 +1,37 @@
 source 'https://rubygems.org'
 
-# TODO: We pretty much just use active record and active mailer, do we need rails?
-gem "mysql2", "~> 0.4.10"
-gem "rails", "~> 4.1"
+gem 'bootsnap'
+gem 'mysql2'
 
-gem "bunny"
-gem "daemons"
-gem "hashie", "~> 3.3"
-gem "migration_comments"
+# We hardly use any of the features of Rails, so lets just load what we need
```

Comment:
Lots of commented out stuff. Can it just be removed?

Reply:
Yes, was mostly left there to direct future developers if they want to reinstate a feature, but may be better of just putting a comment with an example, and a pointer to the rails documentation.